karma/adapter.js

```diff
@@ -4,6 +4,6 @@
 __karma__.start = function() {};
 
 const context = require.context('../../../test');
-global.metafiles
+global.metaconfig.files
   .map(file => '.' + file.slice(file.indexOf('/')))
```

Comment:
Not sure, perhaps you want `path.join`, `path.basename`? Also, I don't think 'dot' will work on Windows as you want.